mobile/src/main.jsx

```diff
@@ -22,6 +23,8 @@ import Settings from './containers/Settings'
 import { loadState, saveState } from './lib/localStorage'
 import { init } from './lib/cozy-helper'
 
+Raven.config('https://29bd1255b6d544a1b65435a634c9ff67@sentry.cozycloud.cc/2').install()
```

Comment:
It is build as: `https://<key>@sentry.cozycloud.cc/<project>` where `key` is generated from [the settings page](https://sentry.cozycloud.cc/sentry/cozy-files-v3/settings/keys/) and `project` is the ID of the project.

So, it will be always the same until we revoke it and generate a new one.